plugins/modules/iosxr_config.py

```diff
@@ -451,10 +451,24 @@ def main():
         supports_check_mode=True,
     )
 
+    warnings = list()
+    os_version = get_os_version(module)
+    if version.parse(os_version) > version.parse("7.2"):
```

Comment:
In case os_version value is `None` due to some reason this can result in stacktrace. Can modify the condition as
`if os_version and version.parse(os_version) > version.parse("7.2"):`